webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -49,6 +50,7 @@ export type State = {
     error: string | null;
     getMetaDataErr: string | null;
     submitting: boolean;
+    showConfirmModal: boolean | false;
```

Comment:
Why is this `| false` needed?

Reply:
ditto strings above?

Reply:
The `string | null` makes sense, because `this.state.error` has its initial value set to `null` instead of say an empty string. Also, when an error is cleared, the state var gets assigned `null`.

This is convention in the webapp so I think it makes sense to do it the same here.

Reply:
@jfrerich I'm just curious, was it an editor warning that had you put the `boolean | false`?